string action
string actionableBlock
string destinationBlock
int16 timestep
Configuration config
bool goalAchieved
